packages/storybook8/stories/Components/MessageThread/MessageThreadWithRichTextEditor/Docs.mdx

```diff
@@ -6,7 +6,7 @@ import MessageThreadWithRichTextEditorExampleText from '!!raw-loader!./snippets/
 import MessageThreadWithRichTextEditorInlineImagesExampleText from '!!raw-loader!./snippets/WithRichTextEditorInlineImages.snippet.tsx';
 import MessageThreadWithRichTextEditorOnPasteExampleText from '!!raw-loader!./snippets/WithRichTextEditorOnPaste.snippet.tsx';
 
-<Meta of={MessageThreadWithRichTextEditorStories} /># MessageThread using Rich Text Editor
```

Comment:
Instead of deleting this I think it should be kept but on its own line

Reply:
It looks correct with the removal on my local, does it look different now?

Reply:
Without it it doesn't have an `h1` level heading:

![image](https://github.com/user-attachments/assets/bd40de4b-dd17-4776-9756-2d084e01eaf5)


Whereas the other message thread stories do:

![image](https://github.com/user-attachments/assets/f1637e9d-24f2-4107-b9cc-d6436b71f9be)